src/app.js

```diff
@@ -130,11 +130,27 @@ class SAFEApp extends EventEmitter {
   * Helper to lookup a given `safe://`-url in accordance with the
   * convention and find the requested object.
   *
+
   * @param {String} url the url you want to fetch
+  * @param {Object} options options Object
+  *                       range: { array }, start/end of partial content range to retrieve
```

Comment:
I just sent a PR to enhance the documentation for this by defining an object named `WebFetchOptions`.